guides/common/modules/proc_preparing-for-disaster-recovery-with-active-and-passive-project-server-with-external-storage.adoc

```diff
@@ -0,0 +1,36 @@
+[id="preparing-for-disaster-recovery-with-active-and-passive-project-server-with-external-storage"]
+= Preparing for disaster recovery with active and passive {ProjectServer} with external storage
+
+Create your passive {ProjectServer} as a clone of your active {ProjectServer}.
+Make sure that both servers connect to the database content on your network-attached storage (NAS).
+
+.Procedure
+. Replicate the `/var/lib/pulp` and `{postgresql-lib-dir}` directories from the active {ProjectServer} to your NAS.
+// Do we want to keep this generic like the virtualization scenario, or is this a good place to provide example rsync commands or something?
+. Clone your active {ProjectServer}.
+See xref:cloning_satellite_server[].
++
+Keep the source server powered on and power off the new cloned server.
+The source server remains your primary active server, while the clone server becomes the secondary passive server.
+. Determine how you want to attach the database content on the NAS to your passive server:
+* If you mount the storage directly by both your active and passive server, the servers will always see the same, up-to-date content.
```

Comment:
`mount ... by` -> `mount ... on`, otherwise it seems like both of these mountings serve some greater purpose, and not just the purpose of these two servers interacting with the data.

Additionally, `mount by` is almost never used, it's usually `mount on` and less frequently `mount at`.